assets/src/components/detours/detourRouteSelectionPanel.tsx

```diff
@@ -56,8 +56,11 @@ export const DetourRouteSelectionPanel = ({
     <Panel.Body className="d-flex flex-column">
       <Panel.Body.ScrollArea className="d-flex flex-column">
         <section className="pb-3">
-          <h2 className="c-diversion-panel__h2">Choose route</h2>
+          <h2 className="c-diversion-panel__h2" id="choose-route-select">
```

Comment:
Are we worried at all about using `id` here, on the off-chance that there's more than one detour modal in the DOM?

Reply:
Just a little, I could throw in a `useId` call, I just liked keeping the arrow function as a pure return 😭

Reply:
It's ... proooooooobably worth doing that? I guess I don't fully know the implications of what happens if there are duplicate `id`'s, but it feels like the sort of area where a bug will creep in like four years from now and future-us will never find it 🙃 

RIP pure arrow function :(

Reply:
(I disabled auto-merge so that this PR doesn't get surprise-merged while you/we are still thinking about this)

Reply:
[There's a conflict which prevents this from being merged](https://github.com/mbta/skate/pull/2649/commits/3229af55870fd6cbfb9757f8468ce20e74bd2e30)